src/pages/Home.tsx

```diff
@@ -78,27 +101,48 @@ const Home = () => {
   };
 
   return (
-    <div
+    <motion.div
       id="main-content"
-      className="items-center justify-center text-center px-4 py-4 flex flex-col"
+      className="items-center justify-center text-center px-4 py-4 flex flex-col bg-[#ffffffd8]"
```

Comment:
```suggestion
      className="items-center justify-center text-center px-4 py-4 flex flex-col bg-white"
```
Could this just be `bg-white` aswell?

Reply:
This is a slightly 'off-white', i think it has a lower opacity. I can make this a class too though